src/index.js

```diff
@@ -1,5 +1,7 @@
 const toMatchStyledComponentsSnapshot = require('./matchers/toMatchStyledComponentsSnapshot')
+const toHaveStyle = require('./matchers/toHaveStyle')
 const styleSheetSerializer = require('./serializers/styleSheetSerializer')
 
 expect.addSnapshotSerializer(styleSheetSerializer)
 expect.extend({ toMatchStyledComponentsSnapshot })
+expect.extend({ toHaveStyle })
```

Comment:
Also changing `toHaveStyle` to `toHaveStyleRule` while leaving the filename the same. Not sure how you want to split the web/native for now.